framework/encode/api_capture_manager.h

```diff
@@ -127,8 +131,6 @@ class ApiCaptureManager
 
     bool IsTrimHotkeyPressed() { return common_manager_->IsTrimHotkeyPressed(); }
 
-    CaptureSettings::RuntimeTriggerState GetRuntimeTriggerState() { return common_manager_->GetRuntimeTriggerState(); }
```

Comment:
Is this deleted in this PR just for cleanup?

Reply:
More or less yes. Since now there are 2 runtime android options `GetRuntimeTriggerState` didn't serve any great purpose returning just the trigger state. Now `GetRuntimeTriggerEnabled`,  `GetRuntimeTriggerDisabled` and the new `RuntimeWriteAssetsEnabled` and `RuntimeWriteAssetsDisabled` fetch the runtime values themselves